community-membership.md

```diff
@@ -220,6 +219,32 @@ The following apply to the subproject for which one would be an owner.
 - Work with other maintainers to maintain the project's overall health and
   success holistically.
 
+### Becoming a Maintainer
+
+Unless stated otherwise in a SIG charter ratified by the Technical Committee,
+a new maintainer is elected by vote of the existing maintainers of the SIG.
+The vote is officially started when a pull request to add a new maintainer
+is opened, and ends when the pull request is merged. The pull request may be
+merged when the following conditions are met:
+
+- The person being nominated has accepted the nomination by approving the pull request
+- All maintainers have approved the pull request OR a majority of maintainers
```

Comment:
I am not sure I understand the "OR" here. It seems like the right-side of the OR is a sufficient condition and the left-side is superfluous.

Reply:
if all maintainers approve you don't need to wait the full 5 days. should we just wait 5 days no matter what?

Reply:
Ah, I see, there is a difference then. I think what you have is fine.